src/main/java/com/arcbees/website/client/application/ApplicationView.java

```diff
@@ -160,4 +199,32 @@ private boolean isFrench() {
         LocaleInfo currentLocale = LocaleInfo.getCurrentLocale();
         return currentLocale.getLocaleName().equals("fr");
     }
+
+    private void watchMenu() {
+        // Hack to get the real menu state (on hover, links trigger false data with .hover())
```

Comment:
I think it would be better to have private functions than comments, if possible.